src/transport/http-transport.js

```diff
@@ -49,6 +49,7 @@ HttpTransport.prototype.submitRpcRequest = function (rpcObject, errorCallback) {
       if (error.code === "ECONNRESET") error.retryable = true;
       if (error.code === "ECONNREFUSED") error.retryable = true;
       if (error.code === "ETIMEDOUT") error.retryable = true;
+      if (error.code === "ESOCKETTIMEDOUT") error.retryable = true;
       if (error.code === "EAI_AGAIN") error.retryable = true;
```

Comment:
Not related to this PR, but WTF is AI_AGAIN?  "Oh, its just the silly AI again, don't mind it, just try again.